parsl/tests/test_error_handling/test_htex_manager_failure.py

```diff
@@ -0,0 +1,52 @@
+import os
+import signal
+import time
+
+import pytest
+
+import parsl
+from parsl.app.app import python_app
+from parsl.tests.configs.htex_local import fresh_config
+
+
+@pytest.fixture(autouse=True)
+def load_config():
```

Comment:
if you call this `local_config`, it will be magically loaded by the parsl test system - see eg. parsl/tests/test_error_handling/test_retry_handler.py

That's defined in parsl/tests/conftest.py:
```
@pytest.fixture(autouse=True, scope='module')
def load_dfk_local_module( ...
```

Reply:
I need the fixture to have function-level scope since the manager dies in each test iteration.